### DISCUSSION TITLE
About diltiazem PBPK model

### DISCUSSION THREAD

**Discussion by balamo98** [Open-Systems-Pharmacology/Forum#1419] [Questions & Problems] [answered]
Hello,
I tried to search for diltiazem PBPK model but could not find anything.
Did anyone worked on this model using PK-Sim?

yours colin

**Comment by StephanSchaller** [comment, marked as answer]
Hi Colin @balamo98 , no, not so far.
Models for verapamil or diazepam have been developed if this could be of help.

**Reply by balamo98** [reply]
thank you，they will be help
